packages/ui/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM node:18-alpine AS base
+
+# Install dependencies only when needed
+FROM base AS deps
+
+## Set NODE_ENV to something other than development
+ENV NODE_ENV build
+
+## Setup workspaces
+WORKDIR /app
+COPY package*.json ./
+
+## Install dependencies
+COPY ./packages/ui/package*.json ./packages/ui/
+COPY ./packages/config ./packages/config
+RUN apk --no-cache --virtual build-dependencies add python3 make g++
```

Comment:
did you need python?

Reply:
yes, didn't work without it

Reply:
was python in official?

Reply:
no just `libc6-compat` but that didn't work and I got an error saying python was not found